SU2_CFD/include/solvers/CEulerSolver.hpp

```diff
@@ -38,304 +39,276 @@
-  Total_MFR,            /*!< \brief Total Mass Flow Ratio for all the boundaries. */
-  Total_Prop_Eff,       /*!< \brief Total Mass Flow Ratio for all the boundaries. */
-  Total_ByPassProp_Eff, /*!< \brief Total Mass Flow Ratio for all the boundaries. */
-  Total_Adiab_Eff,      /*!< \brief Total Mass Flow Ratio for all the boundaries. */
-  Total_Poly_Eff,       /*!< \brief Total Mass Flow Ratio for all the boundaries. */
-  Total_Custom_ObjFunc, /*!< \brief Total custom objective function for all the boundaries. */
-  Total_CSF,      /*!< \brief Total sideforce coefficient for all the boundaries. */
-  Total_CMx,      /*!< \brief Total x moment coefficient for all the boundaries. */
-  Total_CMx_Prev, /*!< \brief Total drag coefficient for all the boundaries (fixed lift mode). */
-  Total_CMy,      /*!< \brief Total y moment coefficient for all the boundaries. */
-  Total_CMy_Prev, /*!< \brief Total drag coefficient for all the boundaries (fixed lift mode). */
-  Total_CMz,      /*!< \brief Total z moment coefficient for all the boundaries. */
-  Total_CMz_Prev, /*!< \brief Total drag coefficient for all the boundaries (fixed lift mode). */
-  Total_CoPx,     /*!< \brief Total x moment coefficient for all the boundaries. */
-  Total_CoPy,     /*!< \brief Total y moment coefficient for all the boundaries. */
-  Total_CoPz,     /*!< \brief Total z moment coefficient for all the boundaries. */
-  Total_CFx,      /*!< \brief Total x force coefficient for all the boundaries. */
-  Total_CFy,      /*!< \brief Total y force coefficient for all the boundaries. */
-  Total_CFz,      /*!< \brief Total z force coefficient for all the boundaries. */
-  Total_CEff,     /*!< \brief Total efficiency coefficient for all the boundaries. */
-  Total_CMerit,   /*!< \brief Total rotor Figure of Merit for all the boundaries. */
-  Total_CT,       /*!< \brief Total thrust coefficient for all the boundaries. */
-  Total_CQ,       /*!< \brief Total torque coefficient for all the boundaries. */
-  Total_Heat,     /*!< \brief Total heat load for all the boundaries. */
-  Total_MaxHeat,  /*!< \brief Maximum heat flux on all boundaries. */
-  Total_AeroCD,   /*!< \brief Total aero drag coefficient for all the boundaries. */
-  Total_CEquivArea,   /*!< \brief Total Equivalent Area coefficient for all the boundaries. */
-  Total_CNearFieldOF, /*!< \brief Total Near-Field Pressure coefficient for all the boundaries. */
-  Total_CpDiff,       /*!< \brief Total Equivalent Area coefficient for all the boundaries. */
-  Total_HeatFluxDiff, /*!< \brief Total Equivalent Area coefficient for all the boundaries. */
-  Total_MassFlowRate; /*!< \brief Total Mass Flow Rate on monitored boundaries. */
+  *Inflow_MassFlow = nullptr,    /*!< \brief Mass flow rate for each boundary. */
+  *Exhaust_MassFlow = nullptr,   /*!< \brief Mass flow rate for each boundary. */
+  *Inflow_Pressure = nullptr,    /*!< \brief Fan face pressure for each boundary. */
```

Comment:
...
using c++11 default initialisation instead of having to repeat this in all constructors